src/blockchain_db/lmdb/db_lmdb.cpp

```diff
@@ -5963,27 +5964,23 @@ struct service_node_proof_serialized
 {
   service_node_proof_serialized() = default;
   service_node_proof_serialized(const service_nodes::proof_info &info)
-    : timestamp{info.timestamp}, ip{info.public_ip}, storage_port{info.storage_port}, quorumnet_port{info.quorumnet_port},
-      version{info.version[0], info.version[1], info.version[2]}, pubkey_ed25519{info.pubkey_ed25519}
-  {
-    boost::endian::native_to_little_inplace(timestamp);
-    boost::endian::native_to_little_inplace(ip);
-    boost::endian::native_to_little_inplace(storage_port);
-    boost::endian::native_to_little_inplace(quorumnet_port);
-    boost::endian::native_to_little_inplace(version[0]);
-    boost::endian::native_to_little_inplace(version[1]);
-    boost::endian::native_to_little_inplace(version[2]);
-  }
+    : timestamp{native_to_little(info.timestamp)},
+      ip{native_to_little(info.public_ip)},
+      storage_port{native_to_little(info.storage_port)},
+      quorumnet_port{native_to_little(info.quorumnet_port)},
+      version{native_to_little(info.version[0]), native_to_little(info.version[1]), native_to_little(info.version[2])},
+      pubkey_ed25519{native_to_little(info.pubkey_ed25519)}
```

Comment:
I think this might trigger that assertion from boost 1.70 regarding un-convertible types (just going off gut feeling though).

Reply:
Crap, that wasn't supposed to be in there.